termprogressbar/progressbar.go

```diff
@@ -0,0 +1,35 @@
+package termuiprogressbar
+
+import (
+	"github.com/cheggaaa/pb"
+)
+
+type termuiprogressbar struct {
+	progressbar *pb.ProgressBar
+	Visible     bool
+}
+
+func NewProgressBar(total int, visible bool) *termuiprogressbar {
+	pbar := termuiprogressbar{}
```

Comment:
Wouldn't just returning a literal be cleaner?
```golang
return &termuiprogressbar{
	progressbar: pb.New64(int64(total)),
	Visible: visible,
}
```
(Obviously I didn't go through the `make vet lint` cycle…)